lib/crawly/utils.ex

```diff
@@ -139,6 +139,31 @@ defmodule Crawly.Utils do
     end
   end
 
+
+  @doc """
+  Returns a list of known modules which implements Crawly.Spider behaviour
+  """
+  @spec list_spiders() :: [module()]
+  def list_spiders() do
+    Enum.reduce(
+      get_modules_from_applications(),
+      [],
+      fn mod, acc ->
+        try do
```

Comment:
Why is the try/rescue needed here? Its an anti-pattern for swallowing errors.

Reply:
Yes, it is. Honestly, I don't like the current approach for getting spiders. But also, I don't like to introduce something like SPIDERS_DIR option to crawly config. At least not yet. 

I took the approach above from a snippet on the internet. It crashes in some cases for example:
```
09:09:35.756 [debug] Could not get behaviour information for: %UndefinedFunctionError{arity: 1, function: :__info__, message: nil, module: :cowboy_websocket, reason: nil}

09:09:35.832 [debug] Could not get behaviour information for: %UndefinedFunctionError{arity: 1, function: :__info__, message: nil, module: :plug_multipart, reason: nil}
```
When it comes to erlangish deps. However, it works for Crawly.Spider behaviour. Probably in future, we need a better way of finding spiders.

Reply:
I've got a snippet that I use for finding spider modules that is regex based. I'll dig it out from my code base to see if it helps

Reply:
thanks, but maybe lets do it as a separate PR. This approach currently works.